messages/event_subscription.go

```diff
@@ -70,18 +69,15 @@ func (es *eventSubscription) eventSupported(
 		return false
 	}
 
-	// The total number of messages must be
-	// greater of equal to the subscription threshold
-	return totalMessages >= es.details.MinNumMessages
+	return true
```

Comment:
Nitpick, but you can inline this with the previous check:
```go
return messageType == es.details.MessageType
```

Reply:
Done in https://github.com/0xPolygon/go-ibft/pull/39/commits/164c6e7294fc1f7a56f65375306981e97ebc4e08